control/src/lib.rs

```diff
@@ -641,7 +657,7 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 
-	fn remove(org_id: T::Hash, account: T::AccountId) -> DispatchResult {
+	fn remove_org_member(org_id: T::Hash, account: T::AccountId) -> DispatchResult {
```

Comment:
Let's finish our long discussion about function names right here.
I'd stick to "do_{extrinsic_name}" format.
Example FRAME Uniques: https://github.com/paritytech/substrate/blob/master/frame/uniques/src/functions.rs#L25

Or, please, suggest other options.

Reply:
Let's use `do_...` naming.